op-guide/configuration.md

```diff
@@ -124,45 +116,37 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: "new"
 + Note: this flag will be deprecated later.
 
-### --port
+### --addr
 
-+ the listening port for client traffic
-+ default: 1234
-+ The listening address is `0.0.0.0:${port}`.
++ the listening address for client traffic
++ default: "0.0.0.0:1234"
 + Note: this flag will be deprecated later.
 
-### --advertise-port
+### --advertise-addr
 
-+ the server advertise port for client traffic
-+ default: ${port}
-+ If you use Docker to start PD, and use different ports to map the listening port, you must set this flag. For example, if you use `-p 11234:1234`, the advertise port must be 11234 through which the client can connect to PD.
++ the server advertise address for client traffic
++ default: ${addr}
++ If the client cannot connect to PD through the default listening address, you must manually set the advertise address explicitly.
 + Note: this flag will be deprecated later.
 
-### --http-port
+### --http-addr
 
-+ the server http port
-+ default: 9090
++ the server http address
++ default: "0.0.0.0:9090"
 + Note: this flag will be deprecated later.
 
 ## TiKV
 
-### -P, --port
-
-+ the server listening port for outer traffic
-+ default: 20160
-+ The listening address is `0.0.0.0:${port}`.
-
-### --advertise-port
+### -A, --addr
 
-+ the server advertise port for outer traffic
-+ default: ${port}
-+ If you use Docker to start TiKV, and use different ports to map the listening port, you must set this flag. For example, if you use `-p 20161:20160`, the advertise port must be 20161 through which the client can connect to TiKV.
++ the server listening address
++ default: "0.0.0.0:20160"
 
-### -H, --host
+### --advertise-addr
 
-+ the server host
-+ default: "127.0.0.1"
-+ You must set this flag so that it can be connected from outside. You can only ignore it in the standalone mode.
++ the server advertise address for client traffic.
++ default: ${addr}
++ + If the client cannot connect to TiKV through the default listening address, you must manually set the advertise address explicitly.
```

Comment:
I think two adverbs are fine. My understanding is: "manually" means the user has to set the flag by himself; "explicitly" means it has to be set and it doesn't have to be manually.  If this is not the case,  maybe we need to state it as you must set the advertise address explicitly.